runtime/errors.go

```diff
@@ -72,13 +84,23 @@ func HTTPError(ctx context.Context, mux *ServeMux, marshaler Marshaler, w http.R
 
 // DefaultHTTPErrorHandler is the default error handler.
 // If "err" is a gRPC Status, the function replies with the status code mapped by HTTPStatusFromCode.
+// If "err" is a HTTPStatus, the function replies with the status code provide by that struct. This is
+// intended to allow passing through of specific statuses via the function set via WithRoutingErrorHandler
+// for the ServeMux constructor to handle edge cases which the standard mappings in HTTPStatusFromCode
+// are insufficient.
```

Comment:
Had to read this sentence three times, but I think we accidentally a word here.
```suggestion
// are insufficient for.
```